packages/builder/src/pages/builder/portal/users/users/[userId].svelte

```diff
@@ -206,7 +206,9 @@
     if (!user?._id) {
       $goto("./")
     }
-    tenantOwner = await users.tenantOwner($auth.tenantId)
+    // Just take the first app because the account holder will belong to all apps.
+    const appId = availableApps[0].devId
```

Comment:
what happens if there's no apps?

Reply:
![Screenshot 2024-10-28 at 10 44 51](https://github.com/user-attachments/assets/db39fa66-003e-41a1-83fa-e3b451221d26)

Currently you can't get into a valid state without at least one app. When you delete your last app, you are presented with the create first app flow.

If you do manage to get into a state with no apps, it throws errors in other parts of the platform.

Reply:
Refactored to not rely on appId.